<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weatherGUI</class>
 <widget class="QWidget" name="weatherGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>36</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>城市</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>36</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>县区</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>昆山</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>省份</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>江苏</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>苏州</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser13">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser14">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser23">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser22">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser24">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser21">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser33">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser32">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser34">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser31">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser43">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser42">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser44">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser41">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser53">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser52">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser54">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser51">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser63">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser62">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser64">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>600</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser61">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>600</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser74">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>650</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser73">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>650</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser72">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>650</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser71">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>650</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Devanagari</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>咖喱怪天气预报</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>天气</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>330</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>风力</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>330</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
